### DISCUSSION TITLE
DataGrid OnSelectedRowStyling

### DISCUSSION THREAD

**Discussion by jklongwith** [Megabit/Blazorise#2028] [General]
In version 0.9.2, I was able to assign a Class and a Background on a selected row using OnSelectedRowStyling. In version 0.9.3, this no longer works. Any chance this is being fixed? Thank you.

**Comment by stsrki** [comment]
I just tried it and the Class and Background gets applied. But the problem is that other CSS styles have a higher priority and they are _winning_ over your applied styles.

To make it work you can try with more specific CSS selectors for your class names. eg. instead of `.bg-primary{}` have `tr td.bg-primary{}`.

**Reply by jklongwith** [reply]
Thank you. After working around with it, I figured it out. I appreciate your help.
